README.md

```diff
@@ -1,29 +1,33 @@
 # SilverStripe DynamoDB
 
-[![Build Status](https://travis-ci.org/silverstripe/silverstripe-dynamodb.svg?branch=master)](https://travis-ci.org/silverstripe/silverstripe-dynamodb)
+[![CI](https://github.com/silverstripe/silverstripe-dynamodb/actions/workflows/ci.yml/badge.svg)](https://github.com/silverstripe/silverstripe-dynamodb/actions/workflows/ci.yml)
```

Comment:
Note that this is wrong in the preview because currently there _is_ no github actions in this repo. It'll fix itself after this PR is merged.